.readthedocs.yaml

```
@@ -0,0 +1,13 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  commands:
+    - mkdir --parents $READTHEDOCS_OUTPUT/html/
+    - pip install nox
+    - nox --session build_docs
+    - cp --recursive site/* $READTHEDOCS_OUTPUT/html/
```

I think it's better to update `noxfile.py` to optionally accept a positional CLI argument for the target folder. This way, we could make the site generator output directly into the desired target.
```suggestion
    - nox --session build_docs -- "${READTHEDOCS_OUTPUT}"/html
```